### ISSUE TITLE
Web interface not working

### ISSUE THREAD

**Issue by tractor137** [springfall2008/batpred#1451]
**Describe the bug**
The web app works from within the predbat add-on but not on via homeassistant.local:5052

**Expected behavior**

**Predbat version**
8.4.6

**Environment details**
5kw inv 9.5kwh battery
predbat add-on

**Screenshots**
If applicable, add screenshots to help explain your problem. The most useful ones can be your battery chart, the Predbat HTML plan and your current settings in HA.

<img width="1408" alt="Screenshot 2024-09-09 at 21 57 21" src="https://github.com/user-attachments/assets/31a1eec2-3e8d-40e3-a221-40eb9aa484aa">

<img width="1432" alt="Screenshot 2024-09-09 at 21 58 34" src="https://github.com/user-attachments/assets/afbf1182-cf94-4ed1-a4f2-47c03bd97492">

<img width="1691" alt="Screenshot 2024-09-09 at 21 59 07" src="https://github.com/user-attachments/assets/32487195-fbb3-4be2-822f-9ca4af644266">

**Log file**
[log.txt](https://github.com/user-attachments/files/16936618/log.txt)

**Comment by matttheotter** [comment]
@tractor137 I assumed it was moved to ingress so is only visible from within HASS now, @springfall2008 is that the case?

**Comment by tractor137** [comment]
I was just going off the guidance in the documentation regarding output data. Interestingly it worked at v8.4.0 (which was the first version I had used after installing the predbat addon) but then as soon as it updated (I think to 8.4.2) it didn’t work and hasn’t since

**Comment by matttheotter** [comment]
The only thing I remember is a note in [v8.4.4](https://github.com/springfall2008/batpred/releases/tag/v8.4.4):

> Note you may have to uninstall and install again the add on if the web interface doesn't work, you don't need to edit your apps.yaml or move any files around they will be safe.

**Comment by tractor137** [comment]
Yes I saw that too and this was what I did just before posting here

**Comment by springfall2008** [comment]
This is as expected, I'm updating the documentation. The direct access to 5052 is a security risk as there is no login.

**Comment by springfall2008** [comment]
As this ticket is old I'm going to close it, please re-open if the issue is there on latest.
